mypy.ini

```diff
@@ -42,7 +42,7 @@ disallow_subclassing_any = True
 disallow_untyped_calls = False
 disallow_untyped_decorators = True
 disallow_untyped_defs = True
-warn_return_any = False
+warn_return_any = True
 warn_unreachable = False
```

Comment:
I'm trying to flip `warn_unreachable` to True as well, but there are issues with the type annotation of `to_param_dict_string()` in `ToolParameter` and its subclasses.
If anyone would like to give it a shot, I'd appreciate the help!